gcp_bolt_parameters.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Looks like unwanted file also got committed, make sure its removed before merge.

Reply:
Intentionally kept the file , just to know the variables about the parameters that need to be passed while provisioning vm on gcp via terraform.

Reply:
Let me remove it for sure